examples/framework-alpine/src/pages/index.astro

```diff
@@ -25,7 +25,6 @@ import Counter from "../components/Counter.astro";
 		<!-- Load AlpineJS on the page -->
 		<script>
 			import Alpine from "alpinejs";
-			window.Alpine = Alpine;
```

Comment:
In the end, I decided to just remove this and not deal with explaining to the user how to augment the Window interface and stuff in this example as it felt too off-topic

This line is optional and not needed for Alpine to work, adding the `@types/alpinejs` fix types in all other occasions